docs/en/observability/aws-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,99 @@
+
+[[aws-elastic-serverless-forwarder]]
+= Elastic Serverless Forwarder
+
+// should we expand titleabbrev to AWS or Amazon Web Services Elastic Serverless Forwarder? What is correct nomenclature?
+
+++++
+<titleabbrev>AWS Elastic Serverless Forwarder</titleabbrev>
+++++
+
+:keywords: serverless
+:description: The Elastic Serverless Forwarder is an Amazon Web Services ({aws}) Lambda function that ships logs from your {aws} environment to Elastic.
+
+The Elastic Serverless Forwarder is an Amazon Web Services ({aws}) Lambda function that ships logs from your {aws} environment to Elastic.
+
+[discrete]
+[[aws-serverless-forwarder-overview]]
+= Overview
+The Elastic Serverless Forwarder can forward {aws} data to self-managed or cloud-hosted Elastic environments. It supports the following inputs:
+
+- Amazon S3 SQS (Simple Queue Service) event notifications
+- Amazon Kinesis Data Streams
+- Amazon CloudWatch Logs subscription filters
+- Direct Amazon SQS message payload
+
+[role="screenshot"]
+image::aws-serverless-lambda-flow.png[AWS Lambda flow]
+
+The Lambda deployment sets up an SQS queue automatically. This is used to trigger a new function invocation that Lambda uses to continue from exactly where the last function run was terminated. By default a Lambda function runs for a maximum of 15 minutes. When processing event data there’s a possibility that the function may be exited by {aws} in the middle of processing. The function handles this scenario gracefully by keeping track of the last offset processed.
+
+Any exception or fail scenarios related to ingestion are handled by Lambda gracefully using a replay queue. Data in the replay queue is stored as individual events. Lambda keeps track of any failed events and writes them to a replay queue that can then be consumed by adding an additional SQS trigger via Lambda.
+
+You can use the <<sample-s3-config-file,config.yaml>> file to configure the service for each input type, including information such as SQS queue ARN and {es} connection details. You can create multiple input sections within the configuration file to map different inputs to specific log types.
+
+The Lambda function also supports writing directly to an index, alias, or custom data stream. This enables existing {es} users to re-use index templates, ingest pipelines, or dashboards that are already created and connected to business processes.
+
+The best way to get started is to install the appropriate https://docs.elastic.co/en/integrations[integrations] via {kib}. These integrations include pre-built dashboards, ingest node configurations, and other assets that help you get the most out of the data you ingest. The integrations use https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html[data streams] with specific https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme[naming conventions] that provide you with more granular controls and flexibility on managing data ingestion.
+
```

Comment:
Add assets after integration.
The best way to get started is to install the appropriate https://docs.elastic.co/en/integrations[integrations] **assets** via {kib}. These integrations **assets** include pre-built dashboards, ingest node configurations, and other assets that help you get the most out of the data you ingest. The integrations use https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html[data streams] with specific https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme[naming conventions] that provide you with more granular controls and flexibility on managing data ingestion.